integrations/extensions/starter-kits/language-model-openai/README.md

```diff
@@ -27,50 +28,105 @@ This starter kit requires that you use the [new Watson Assistant](https://cloud.
+
+### Build and add a custom extension for Azure OpenAI
+
+1.  In your assistant, on the **Integrations** page, click **Build custom extension** and use the [Azure OpenAPI specification file](./azure-openai-openapi.json) to build a custom extension named `Azure OpenAI`. For general instructions on building any custom extension, see [Building the custom extension](/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
+
+1.  After you build the Azure OpenAI extension and it appears on your **Integrations** page, click **Add** to add it to your assistant. For general instructions on adding any custom extension, see [Adding an extension to your assistant](/docs/watson-assistant?topic=watson-assistant-add-custom-extension).
+
+1.  In **Authentication**, choose **API key auth**. Copy and paste the Azure OpenAI API key that you created [in this section](#create-an-api-key-for-azure-openai) into the **API key** field.
+1.  Specify your server variables for a valid URL for requests. For example, to complete this url `https://{endpoint}/openai`, you add the name of your Azure OpenAI resource that you created have created with Azure OpenAI to complete this endpoint `{your-azure-resource-name}.openai.azure.com`.
+
+If you need any capabilities that are in the documentation but not in the Azure OpenAPI specification provided in this starter kit, feel free to add them. The specification in the kit is intended to be a simple example of how to get started, not a comprehensive encoding of everything that the API can do.
+
+For more information about the extension parameters, see the [Azure OpenAI API Reference](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference).
+
+### Upload sample actions
+
+Use **Actions Global Settings** to upload the [`azure-openai-actions.json`](./azure-openai-actions.json) file in this kit to your assistant. For more information, see [Uploading](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import). You may need to refresh the action Preview chat after uploading in order to get all the session variables initialized before these actions will work correctly.
+
+### Session variables
+
+These are the session variables used in this example. Most of values are set in the process of setting up the starter kit so you do not need to set them yourself unless you want to make changes. You must, however, set the azure deployment id variables `azure_deployment_id_chat` and `azure_deployment_id_completions` to the respective values of the models that you have deployed with Azure OpenAI.
+
+- `azure_api_version` The version of the Azure OpenAI api spec, defaults to `2023-03-15-preview`.
+- `azure_deployment_id_chat` You **MUST** set this value to be the deployment id of the model (e.g., `gpt-3.5-turbo`) you deployed with Azure OpenAI for chat.
```

Comment:
I would replace
> (e.g., `gpt-3.5-turbo`)
with
> (e.g., `my-gpt-3.5-turbo-deployment`)
to emphasize this is a label that the user defines in Azure.